### ISSUE TITLE
Preview fails on newly created  content

### ISSUE THREAD

**Issue by RobertCopilau** [umbraco/Umbraco-CMS#4578]
How to replicate:
1) Install the SK on a fresh v8 site
2) Create a ContenPage under Home
3) Try to preview it

![{E0D155EC-99E6-4AEA-A841-219285E44775}.png.jpg](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/5b151a952d7cd20011c4533d/125516c66e82c728ace21e0d46f3b1ca07d3d6fcaacaf21afd78a6d97d6702526425fb7f21c8b179e61023542a5b4aab465d125abca7de39baa47c32d5065ebe957f5566c36f44aeec851cba55ec24884a85b6bd8b49c5bd3471b18e9d532c5c5446dda0006ceacc09da52e44ae84754a1163662e27334754f9c17703e26.jpg)

```2019-02-14 12:50:09,023 [P48508/D2/T9] ERROR  Umbraco.Web.UmbracoApplicationBase - An unhandled exception occurred
System.NullReferenceException: Object reference not set to an instance of an object.
   at Umbraco.Web.Models.PublishedContentBase.GetUrl(String culture) in C:\Users\rober\Desktop\Umbraco_Repos\Umbraco-CMS-v8\src\Umbraco.Web\Models\PublishedContentBase.cs:line 89
   at ASP._Page_views_partials_navigation_TopNavigation_cshtml.Execute() in c:\Users\rober\Desktop\V8\views\partials\navigation\TopNavigation.cshtml:line 5
   at System.Web.WebPages.WebPageBase.ExecutePageHierarchy()
   at System.Web.Mvc.WebViewPage.ExecutePageHierarchy()
   at System.Web.WebPages.WebPageBase.ExecutePageHierarchy(WebPageContext pageContext, TextWriter writer, WebPageRenderingBase startPage)
   at Umbraco.Web.Mvc.ProfilingView.Render(ViewContext viewContext, TextWriter writer) in C:\Users\rober\Desktop\Umbraco_Repos\Umbraco-CMS-v8\src\Umbraco.Web\Mvc\ProfilingView.cs:line 25
   at System.Web.Mvc.Html.PartialExtensions.Partial(HtmlHelper htmlHelper, String partialViewName, Object model, ViewDataDictionary viewData)
   at ASP._Page_Views_master_cshtml.Execute() in c:\Users\rober\Desktop\V8\Views\master.cshtml:line 28
   at System.Web.WebPages.WebPageBase.ExecutePageHierarchy()
   at System.Web.Mvc.WebViewPage.ExecutePageHierarchy()
   at System.Web.WebPages.WebPageBase.ExecutePageHierarchy(WebPageContext pageContext, TextWriter writer, WebPageRenderingBase startPage)
   at System.Web.WebPages.WebPageBase.<>c__DisplayClass40_0.<RenderPageCore>b__0(TextWriter writer)
   at System.Web.WebPages.WebPageBase.Write(HelperResult result)
   at System.Web.WebPages.WebPageBase.RenderSurrounding(String partialViewName, Action`1 body)
   at System.Web.WebPages.WebPageBase.PopContext()
   at Umbraco.Web.Mvc.ProfilingView.Render(ViewContext viewContext, TextWriter writer) in C:\Users\rober\Desktop\Umbraco_Repos\Umbraco-CMS-v8\src\Umbraco.Web\Mvc\ProfilingView.cs:line 25
   at System.Web.Mvc.ViewResultBase.ExecuteResult(ControllerContext context)
   at System.Web.Mvc.ControllerActionInvoker.InvokeActionResultFilterRecursive(IList`1 filters, Int32 filterIndex, ResultExecutingContext preContext, ControllerContext controllerContext, ActionResult actionResult)
   at System.Web.Mvc.ControllerActionInvoker.InvokeActionResultFilterRecursive(IList`1 filters, Int32 filterIndex, ResultExecutingContext preContext, ControllerContext controllerContext, ActionResult actionResult)
   at System.Web.Mvc.ControllerActionInvoker.InvokeActionResultFilterRecursive(IList`1 filters, Int32 filterIndex, ResultExecutingContext preContext, ControllerContext controllerContext, ActionResult actionResult)
   at System.Web.Mvc.ControllerActionInvoker.InvokeActionResultFilterRecursive(IList`1 filters, Int32 filterIndex, ResultExecutingContext preContext, ControllerContext controllerContext, ActionResult actionResult)
   at System.Web.Mvc.ControllerActionInvoker.InvokeActionResultWithFilters(ControllerContext controllerContext, IList`1 filters, ActionResult actionResult)
   at System.Web.Mvc.Async.AsyncControllerActionInvoker.<>c__DisplayClass3_1.<BeginInvokeAction>b__1(IAsyncResult asyncResult)
   at System.Web.Mvc.Async.AsyncControllerActionInvoker.EndInvokeAction(IAsyncResult asyncResult)
   at System.Web.Mvc.Controller.<>c.<BeginExecuteCore>b__152_1(IAsyncResult asyncResult, ExecuteCoreState innerState)
   at System.Web.Mvc.Async.AsyncResultWrapper.WrappedAsyncVoid`1.CallEndDelegate(IAsyncResult asyncResult)
   at System.Web.Mvc.Controller.EndExecuteCore(IAsyncResult asyncResult)
   at System.Web.Mvc.Async.AsyncResultWrapper.WrappedAsyncVoid`1.CallEndDelegate(IAsyncResult asyncResult)
   at System.Web.Mvc.Controller.EndExecute(IAsyncResult asyncResult)
   at System.Web.Mvc.MvcHandler.<>c.<BeginProcessRequest>b__20_1(IAsyncResult asyncResult, ProcessRequestState innerState)
   at System.Web.Mvc.Async.AsyncResultWrapper.WrappedAsyncVoid`1.CallEndDelegate(IAsyncResult asyncResult)
   at System.Web.Mvc.MvcHandler.EndProcessRequest(IAsyncResult asyncResult)
   at System.Web.HttpApplication.CallHandlerExecutionStep.System.Web.HttpApplication.IExecutionStep.Execute()
   at System.Web.HttpApplication.ExecuteStepImpl(IExecutionStep step)
   at System.Web.HttpApplication.ExecuteStep(IExecutionStep step, Boolean& completedSynchronously)```

**Comment by zpqrtbnk** [comment]
Have you been able to repro it or is it over now? If the error does happen in `PublishedContentBase` at line 89... it would mean that some core elements of Umbraco were missing - like, it did not initialize at all or something. Very weird.

**Comment by RobertCopilau** [comment]
I'll try again today with today latest build.

**Comment by zpqrtbnk** [comment]
Reproduced

**Comment by zpqrtbnk** [comment]
Should be fixed by 6a5f8c0ab8
